### PR TITLE
analyze: slightly reword PrivateTmp= message

### PR DESCRIPTION
**Pull request by poettering**
Apparently there way confusion about "does not apply". Let's say "is not
appropriate".

Fixes: #13095

### CODE REVIEW

**Review by bluca — APPROVED** [review]
LGTM

**Comment by yuwata** [comment]
Typo in commit message: PrivatTmp= -> PrivateTmp=

**Comment by poettering** [comment]
oops, sorry, for some reason @yuwata's comment only popped up here once i hit the merge button, even though the merge apparently took place after the comment... i guess some weird latency in github code?

**Pull request merged**
